test/pkg/gitea/test.go

```diff
@@ -82,7 +87,12 @@ func TestPR(t *testing.T, topts *TestOpts) func() {
 	assert.NilError(t, err)
 	topts.GitCloneURL = url
 	PushFilesToRefGit(t, topts, entries, topts.DefaultBranch)
-
+	// in case of some test we create resources before creating pull req
+	// for eg. ClusterTask test, pipeline is getting executed before ClusterTask
+	// is getting created, this is to add some delay in case of that test
+	if topts.WaitForResourceCreation {
```

Comment:
in case of clustertask test, it was always failing for me locally and in CI too
pipelinerun was getting executed before clustertask was created so added a sleep for that test

Reply:
it's weird that it went green sometime !